src/Infocom.php

```diff
@@ -819,11 +819,11 @@ public static function showDisplayLink($itemtype, $device_id)
     /**
      * Calculate amortization values
      *
-     * @param number $value       Purchase value
-     * @param number $duration    Amortise duration
-     * @param string $fiscaldate  Begin of fiscal excercise
-     * @param number $buydate     Buy date
-     * @param number $usedate     Date of use
+     * @param number        $value       Purchase value
+     * @param number        $duration    Amortise duration
+     * @param string        $fiscaldate  Begin of fiscal excercise
+     * @param number|string $buydate     Buy date
+     * @param number|string $usedate     Date of use
```

Comment:
These dates are not supposed to b passed as numbers. The previously documented type was completely wrong.

```suggestion
     * @param string $buydate     Buy date
     * @param string $usedate     Date of use
```